packages/rag/src/tools/BEDROCK_MANAGED_KB.md

```
@@ -0,0 +1,59 @@
+  region: 'us-east-1',
+});
+
+const agent = new Agent({
+  name: 'research-agent',
+  tools: { kb: kbTool },
+  instructions: 'Use the knowledge base to answer questions.',
+});
+```
+
+## Configuration
+| Variable | Description | Default |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)